Reconstruction/RecCalorimeter/src/components/SingleCaloTowerTool.cpp

```diff
@@ -1,183 +0,0 @@
-#include "SingleCaloTowerTool.h"
```

Comment:
Do we want to remove completely this class? It's not going to be useful e.g. for debugging? Or simple tests? (Do you think it's going to create problems to keep both parts updated).
Because if so maybe the other class should be called simply CaloTowerTool (without Combined)? - just an idea

Reply:
Yes, I prefer to completely remove this tool. It is quite straight forward to use the general tool with only one collection as it is done in runEcalSimple_ReconstructionSW_noiseFromFile.py. I renamed the tool as you suggested.

Reply:
ok, thanks!